docs/design/node-image.md

```diff
@@ -35,6 +35,6 @@ each "node" container with [kubeadm][kubeadm].
 [base image]: ./base-image.md
 [build package]: ./../../pkg/build
 [cluster package]: ./../../pkg/cluster
-[docker image archives]: https://docs.docker.com/engine/reference/commandline/save/
+[container image archives]: https://docs.docker.com/engine/reference/commandline/save/
```

Comment:
unless we have a generic reference for this instead, it seems like renaming this and then linking to docker is not particularly helpful.